src/modules/governance/governance.contract.resolver.ts

```diff
@@ -1,62 +1,62 @@
 import { Parent, ResolveField, Resolver } from '@nestjs/graphql';
 import { GovernanceAbiService } from './services/governance.abi.service';
-import { GovernanceContract } from './models/governance.contract.model';
+import { EnergyContract } from './models/energy.contract.model';
 import { GovernanceProposal } from './models/governance.proposal.model';
 
-@Resolver(() => GovernanceContract)
-export class GovernanceContractResolver {
+@Resolver(() => EnergyContract)
+export class EnergyContractResolver {
```

Comment:
please rename with `governance` in name. It can be confusing with the actual energy contract resolver

Reply:
renamed